open_rarity/models/collection.py

```diff
@@ -67,14 +66,51 @@ def __init__(
         tokens: list[Token],
         name: str | None = "",
     ):
+        self._tokens = tokens
```

Comment:
this file's changes was actually for: https://github.com/OpenRarity/open-rarity/pull/39
but i accidentally pushed to wrong PR and merged the other one. cc @theelderbeever

Reply:
This is to disallow users to reset @tokens

Reply:
Great, acknowledged